src/averageAges.js

```diff
@@ -14,8 +14,22 @@
  *
  * @return {number}
  */
+const DIVIDE_NUM = 100;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
+  let filtredMen = people.filter(person => person.sex === 'm');
+
+  if (century !== undefined) {
+    filtredMen = filtredMen
+      .filter(person => Math.ceil(person.died / DIVIDE_NUM) === century);
+  }
+
+  const calculateTotalAge = filtredMen
+    .reduce((sum, person) => sum + (person.died - person.born), 0);
+
+  const calculateAverageAge = calculateTotalAge / filtredMen.length;
```

Comment:
Minor, but the name should be `calculatedAverageAge` or similar
For now, it sounds like a function name

And most probably you don't even need this - just return the result od division